index.html

```diff
@@ -1,8 +1,36 @@
-Annah Nyarangi
-About
-I am a beginner programmer, currently working on JavaScript. I look forward to learning more. Currently working as a nursing assistant and hoping to make a career switch from healthcare to Information Technology.
-Experience
-2015 - Current 
-Nursing Assistant, GI Acute Care
-Connect
-https://github.com/anyarangi
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+           Annah Nyarangi code the dream JavaScript project.
+        </title>
+        <meta charset="UTF-8">
+        <meta name="keywords" content="HTML, CSS, JavaScript">
+        <meta name="author" content="Annah Nyarangi">
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <section> 
+                <h1> Annah Nyarangi</h1>
+                <h2>About</h2>
+                    <p>I am a beginner programmer, currently working on JavaScript; I look forward to learning more. <br>
+                        Currently working as a nursing assistant 
+                        and hoping to make a career switch from healthcare to Information Technology.
+                    </p>    
+        </section>
+        <section> 
+                <h2>Experience</h2>
+                    <ul>
+                        <li>2015 - Current </li>
+                        <li>Nursing Assistant, GI Acute Care </li>
+                    </ul>
+                    </p>
+        </section>
+        <section>
+                <h2>Connect</h2>
+                <a href="https://github.com/anyarangi" target="_blank">Visit my GitHub page!</a>
+                
+        </section>
+</body>
+</html>
```

Comment:
All (opening and closing) tags should be aligned. And be deeply nested/indented compared with their parent elements.